src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -0,0 +1,154 @@
+--[[ Library for cards that seal tokens
+This file is used to add sealing option to cards' context menu.
+Valid options (set before requiring this file):
+SHOW_READ_BAG       -- boolean
+SHOW_SINGLE_RELEASE -- boolean
+SHOW_MULTI_RELEASE  -- number (amount of tokens to release at once)
+VALID_TOKENS        -- table ([tokenName] = true)
+INVALID_TOKENS      -- table ([tokenName] = true)]]
+
+local blessCurseManagerApi = require("chaosbag/BlessCurseManagerApi")
+local tokenArrangerApi = require("accessories/TokenArrangerApi")
+
+function onSave() return JSON.encode(sealedTokens) end
+
+function onLoad(savedData)
+  sealedTokens = JSON.decode(savedData) or {}
+  ID_URL_MAP = Global.getTable("ID_URL_MAP")
+  generateContextMenu()
+end
+
+-- builds the context menu
+function generateContextMenu()
+  self.clearContextMenu()
+
+  -- only show this for cards that need a dynamic list of tokens (for example 'Unrelenting')
+  if SHOW_READ_BAG then
+    self.addContextMenuItem("Update list", generateContextMenu)
```

Comment:
If this isn't an intensive operation you could update the list in onHover(), which would remove the need for a manual user update.

Reply:
Great suggestion! Updated